Lib/test/test_tools/test_msgfmt.py

```diff
@@ -121,10 +124,59 @@ def test_nonexistent_file(self):
         assert_python_failure(msgfmt, 'nonexistent.po')
 
 
+class Test_multi_input(unittest.TestCase):
+    """Tests for the issue https://github.com/python/cpython/issues/79516
+        msgfmt.py shall accept multiple input files
```

Comment:
```suggestion
        msgfmt.py accepts multiple input files
```

If it's merged it's testing an implemented feature, not a POC :-)